appcenter-link-scripts/src/android/index.js

```diff
@@ -75,11 +75,13 @@ module.exports = {
         const gradlePath = 'android/app/build.gradle';
         let gradleContent = fs.readFileSync(gradlePath, 'utf-8');
         gradleContent.split('\n').forEach((line) => {
-            if (lines[line]) {
-                gradleContent = gradleContent.replace(line, '');
-            }
-            if (line.match(/^\s*compile project.*appcenter.*$/)) {
-                lines[line] = true;
+            let match = line.match(/^\s*(compile|implementation) (project.*appcenter.*)$/);
```

Comment:
eslint error "78:17  error  'match' is never reassigned. Use 'const' instead  prefer-const" from VSTS build. Please change `let` to `const`.